.github/workflows/continuous-integration.yml

```diff
@@ -61,11 +61,11 @@ jobs:
 
       - name: Publish to Chromatic
         if: '!github.event.pull_request.head.repo.fork'
-        uses: chromaui/action@v10
+        uses: chromaui/action@v11
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
-          storybookBuildDir: 'webview/storybook-static'
+          workingDir: 'webview'
```

Comment:
PTAL at the docs and see if this is actually the correct thing to do.

Reply:
Maybe I'm misunderstanding something, but it looks like we can remove the `Build Storybook` step. Looking at [the docs](https://www.chromatic.com/docs/github-actions/#run-chromatic-on-monorepos), if we're using `workingDir`, we don't need to build storybook ourselves.

Reply:
@julieg18 We could skip the `Build Storybook` step, but I think that if we do, we won't have the separate check anymore with the Storybook link.

Reply:
We might need `storybookBaseDir` now as well as `storybookBuildDir`. It seems like a bug on their part. They validate the option, but they probably shouldn't if `storybookBuildDir` is specified. Unless they need extra config in the base directory. `workingDir` does seem do make things simpler, because it lets chromatic do everything. If we can output the storybook link with this step and get rid of the previous one as @julieg18 mentioned, that can work.

Reply:
Everything seems to work fine without the build step.

Reply:
PLMK if I missed something, otherwise feel free to merge 🙏🏻.